CMakeLists.txt

```diff
@@ -396,9 +396,9 @@ IF (INSTALL_LAYOUT)
             set(SUSE_VERSION_NUMBER "${CMAKE_MATCH_1}")
         ENDIF ()
         if (${SUSE_VERSION_NUMBER} EQUAL 12)
-            SETA(CPACK_RPM_columnstore-engine_PACKAGE_REQUIRES "expect" "boost-devel >= 1.54.0" "snappy" "jemalloc" "net-tools MariaDB-server" PARENT_SCOPE)
+            SETA(CPACK_RPM_columnstore-engine_PACKAGE_REQUIRES "expect" "boost-devel >= 1.54.0" "snappy" "jemalloc" "net-tools" "MariaDB-server" "python3" PARENT_SCOPE)
```

Comment:
Not sure if these code paths are ever used, but won't hurt to add dependency here as well

Reply:
They are used unfortunately. We need to drop them in favor of those in server's storage/columnstore/CMakeLists.txt for RPM and server's debian/control.